certora/harnesses/AccessManagerHarness.sol

```diff
@@ -0,0 +1,142 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import "../patched/access/manager/AccessManager.sol";
+
+contract AccessManagerHarness is AccessManager {
+    constructor(address initialAdmin) AccessManager(initialAdmin) {}
+
+    // bug introduction: inverse check-effect
```

Comment:
We should remove this, the code will become out of sync with the contract.

Reply:
done